src/hooks/useRelatedVenues.tsx

```diff
@@ -3,14 +3,19 @@ import { useFirestore, useFirestoreCollectionData } from "reactfire";
 
 import { ALWAYS_EMPTY_ARRAY } from "settings";
 
-import { AnyVenue } from "types/venues";
+import { AnyVenue, SpaceSlug } from "types/venues";
 
 import { withIdConverter } from "utils/converters";
 import { WithId } from "utils/id";
 import { findSovereignVenue } from "utils/venue";
 
 import { isEmpty } from "./useFirestoreConnect";
 
+export type FindVenueInRelatedVenuesArgs = {
```

Comment:
```suggestion
export type FindVenueInRelatedVenuesOptions = {
```

Reply:
Resolved in 72a644e7

Reply:
I was hoping github would update. Not even an edit fixed it.

Trying here: 72a644e707f4fe83808d256dee622312e02a89d9